components/ai_chat/core/browser/constants.cc

```diff
@@ -88,7 +88,15 @@ base::span<const webui::LocalizedString> GetLocalizedStrings() {
       {"gotItButtonLabel", IDS_CHAT_UI_GOT_IT_BUTTON_LABEL},
       {"pageContentTooLongWarning", IDS_CHAT_UI_PAGE_CONTENT_TOO_LONG_WARNING},
       {"errorConversationEnd", IDS_CHAT_UI_CONVERSATION_END_ERROR},
-      {"leoSettingsTooltipLabel", IDS_CHAT_UI_LEO_SETTINGS_TOOLTIP_LABEL}};
+      {"leoSettingsTooltipLabel", IDS_CHAT_UI_LEO_SETTINGS_TOOLTIP_LABEL},
+      {"summarizePageButtonLabel", IDS_CHAT_UI_SUMMARIZE_PAGE},
+      {"welcomeGuideTitle", IDS_CHAT_UI_WELCOME_GUIDE_TITLE},
+      {"siteHelpCardTitle", IDS_CHAT_UI_SITE_HELP_CARD_TITLE},
+      {"siteHelpCardDesc", IDS_CHAT_UI_SITE_HELP_CARD_DESC},
+      {"siteHelpCardDescWithAction", IDS_CHAT_UI_SITE_HELP_CARD_WITH_ACTION},
+      {"chatCardTitle", IDS_CHAT_UI_CHAT_CARD_TITLE},
+      {"chatCardDesc", IDS_CHAT_UI_CHAT_CARD_DESC},
```

Comment:
One string is prefixed `WelcomeGuide` but the others aren't - shouldn't we be consistent so it's obvious these strings are for that specific view only? Same in the .grdp

Reply:
done